### ISSUE TITLE
[bug] ServiceMonitor labels doesn't marshal with umbrella chart

### ISSUE THREAD

**Issue by yalattas** [conduktor/conduktor-public-charts#128]
code
```
metrics:
      enabled: true
      serviceMonitor:
        enabled: true
        labels:
          release: Prometheus
```

![Screenshot from 2024-11-13 02-25-15](https://github.com/user-attachments/assets/e68ede97-70ba-4fb8-b305-195df6408a55)

I am using umbrella chart for argocd to apply

```
one or more objects failed to apply, reason: error when patching "/dev/shm/299186659": ServiceMonitor.monitoring.coreos.com "kafka-gui-monitor" is invalid: [metadata.labels: Invalid value: "error unmarshaling JSON: while decoding JSON: json: cannot unmarshal string into Go value of type map[string]interface {}": must be no more than 63 characters, metadata.labels: Invalid value: "error unmarshaling JSON: while decoding JSON: json: cannot unmarshal string into Go value of type map[string]interface {}": a valid label must be an empty string or consist of alphanumeric characters, '-', '_' or '.', and must start and end with an alphanumeric character (e.g. 'MyValue', or 'my_value', or '12345', regex used for validation is '(([A-Za-z0-9][-A-Za-z0-9_.]*)?[A-Za-z0-9])?')]. Retrying attempt #4 at 11:26PM.
```

If I uncomment the labels, it applied successfully, Yet, my prometheus can't discover the serviceMonitor without the `release` label

related to this line
https://github.com/conduktor/conduktor-public-charts/blob/c9d907e0f9340f7bcc4a0ce0dd2aaa0dfcd84b98/charts/console/templates/console/servicemonitor.yaml#L14

**Comment by yalattas** [comment]
```commonLabels:
    release: observability```
    
    doesn't work as well

**Comment by qboileau** [comment]
Thanks for the report, we will investigate this.

**Comment by marnas** [comment]
Hi @yalattas , after doing some work on this I noticed this issue has already been addressed by #122  which was merged on the 17th Oct.

Our latest release `1.13.0` is dated 15th Oct which is why the issue still presents itself, so you should get the fix with our next release.
